parlai/tasks/bot_adversarial_dialog/README.md

```diff
@@ -0,0 +1,16 @@
+Task: Bot-Adversarial Dialogue Dataset
```

Comment:
nit: can we change the folder path from `bot_adversarial_dialog` to `bot_adversarial_dialogue` to be consistent with the spelling?

Reply:
good catch! will fix